src/main/java/de/rabea/Main.java

```diff
@@ -16,7 +24,10 @@ public static void main(String[] args) throws IOException {
 
         ServerSocket serverSocket = new ServerSocket(port);
         Router router = new Router();
-        Logger logger = new Logger();
+
+        MultiLogger logger = new MultiLogger(new ArrayList<>(Arrays.asList(new FileLogger(logFilePath),
```

Comment:
This shouldn't need to be a `MultiLogger` on the left-hand side of the assignment... nobody should know what kind of logger they are talking to...